### ISSUE TITLE
Warning: "Do not run Composer as root/super user"

### ISSUE THREAD

**Issue by alexellis** [itscaro/openfaas-template-php#3]
Hi when testing the template I saw an error from composer, I wondered if we can use a regular user for this step?

```
Do not run Composer as root/super user! See https://getcomposer.org/root for details
```

The other templates now use a multi-stage build and non-root user, perhaps you could look at adding a user called `app` too? See the node template for an example.

**Comment by itscaro** [comment]
@alexellis Yeah, I added a user `app` but for some reasons, I comment out the `USER app` before running composer. I'll try to find some time to test and update the template this week :)

**Comment by stanelyg** [comment]
Am also experiencing the same ,am setting a Laravel project and some folders are missing according to https://laravel.com/docs/5.6/structure so i thought by running the composer will sort the issue any help very new to Laravel
